phonebook-ui/src/app/common/service/user.service.ts

```diff
@@ -0,0 +1,24 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import {User} from "../model/user";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+
+  private host = 'http://localhost:8080/';
+  private registerUrl = `${this.host}api/v1/registration`;
```

Comment:
api/v1/registration -> api/v1/user/registration
the same api/v1/user/confirmation

Reply:
I suggest "v1" to remove!

Reply:
otherwise, in version 2, we will change all links on the frontend and backend!!!